source/host.c

```diff
@@ -942,7 +958,9 @@ void Host_Init (quakeparms_t *parms)
 		SCR_Init ();
 		R_Init ();
 		S_Init ();
+#ifndef __WII__
```

Comment:
Can you make CDAudio_Init blank instead (or similar) so we can remove this ifdef?